### PR TITLE
Add LUKS keyboard issue to the known limitations

### PR DESCRIPTION
**Pull request by erlefloch**
Adding a small paragraph about this issue, as proposed in https://github.com/fedora-silverblue/issue-tracker/issues/3#issuecomment-1637154862

### CODE REVIEW

File: modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -23,6 +23,11 @@ If you are uncertain about this, {variant-name} can also be tested in a virtual
 It is possible to make {variant-name} work for both dual boot and manual partitioning, and some guidance is provided on manual partitioning below.
 However, there are hazards involved in both cases, and you should only attempt to use these features if you have done the necessary research, and are confident that you can overcome any issues that you might encounter.
 
+*The keyboard layout selected during installation is not used for the LUKS unlock screen of {variant-name}.*
+
+Regardless of your selection during the installation process, the keyboard layout available for the LUKS unlock screen will default to `en-US` (English - United States of America).
+There are some workarounds available, as described in the https://bugzilla.redhat.com/show_bug.cgi?id=1890085[bug] opened in 2020 on the Red Hat bugzilla, or on the https://github.com/fedora-silverblue/issue-tracker/issues/3[issue #3] in the Fedora Silverblue issue tracker on GitHub.
```
Comment (travier) [inline modules/ROOT/pages/installation.adoc:29]: Let's directly include the workaround instructions here and link to the references for tracking. Something like:

```suggestion
You can workaround this issue by running the following command on the first boot after the installation:
 rpm-ostree initramfs-etc --track=/etc/vconsole.conf

See https://bugzilla.redhat.com/show_bug.cgi?id=1890085[bug #1890085] (Red Hat Bugzilla) and https://github.com/fedora-silverblue/issue-tracker/issues/3[issue #3] (Fedora Silverblue issue tracker) for details.
```

**Comment by travier** [comment]
Thanks for the PR. Added a suggestion.

File: modules/ROOT/pages/installation.adoc (hunk 1 above)
Comment (erlefloch) [reply modules/ROOT/pages/installation.adoc:29]: good idea !

**Comment by erlefloch** [comment]
You're welcome ! I also added a small practical tip, let me know if it's suitable there.

**Comment by miabbott** [comment]
Thanks for the PR!  Could you squash the commits?  Then it should be good to go.

**Comment by travier** [comment]
I've squashed the commits. Thanks!

**Pull request merged**